source/_integrations/openplantbook.markdown

```diff
@@ -0,0 +1,117 @@
+---
+title: OpenPlantbook
+description: Instructions on how to setup OpenPlantbook with Home Assistant.
+ha_category:
+  - Environment
+ha_iot_class: Cloud Polling
+ha_release: 0.116
+ha_quality_scale: internal
+ha_codeowners:
+  - '@Olen'
+ha_domain: openplantbook
+---
+
+The `openplantbook` component allows you to search and get data from the [OpenPlantbook API](https://open.plantbook.io/) 
+
+## Configuration
+
+The integration is set up using the GUI.  You must have a valid `client_id` and `secret` from OpenPlantbook to set up the integration.
+
+{% configuration %}
+client_id:
+  type: string
+  description: The Client ID from https://open.plantbook.io/apikey/show/
+secret:
+  type: string
+  description: The Client Secret from https://open.plantbook.io/apikey/show/
+{% endconfiguration %}
+
+
+## Examples
+
+Two service calls are added by this integration:
+
+`openplantbook.search` searches the API for plants matching a string. The search result is added to the entity `openplantbook.search_result` with the number of returned results as the `state` and a list of results in the state attributes.
+
+{% raw %}
```

Comment:
You can remove the `raw` and `endraw` tags for this piece of yaml 😉

Reply:
Removed raw-tags for yaml. But kept them for jinja2